core-bundle/contao/elements/ContentAlias.php

```diff
@@ -22,46 +22,24 @@ class ContentAlias extends ContentElement
 	 */
 	public function generate()
 	{
-		$objElement = ContentModel::findByPk($this->cteAlias);
-
-		if ($objElement === null)
+		if ($this->isHidden())
 		{
 			return '';
 		}
 
-		$strClass = static::findClass($objElement->type);
+		$objElement = ContentModel::findByPk($this->cteAlias);
 
-		if (!class_exists($strClass))
+		if ($objElement === null)
 		{
 			return '';
 		}
 
-		if (is_a($strClass, ContentProxy::class, true))
```

Comment:
no I don't think we still need that. But did you manually test keeping CSS class and ID still works, even when using fragments?

Reply:
No. How do I test this? Why don‘t we have a unit tests for this?

Reply:
> No. How do I test this? 

create an alias content element, select another element, add CSS classes to both and see what you get in the front end is what I would do.

> Why don‘t we have a unit tests for this?

I guess because we don't have tests for any old modules or elements?

Reply:
> create an alias content element, select another element, add CSS classes to both and see what you get in the front end is what I would do.

This is what I did. But it does not seem to fulfill the `is_a($strClass, ContentProxy::class, true)` condition, does it?

Reply:
@ausi and I have tested it and the CSS classes are assigned correctly to the alias element.